angular.json

```diff
@@ -497,10 +497,6 @@
               {
                 "command": "nx build-package eslint-plugin"
               },
-              // uncomment when there are eslint-plugin migrations
```

Comment:
`yarn format` keeps changing this. Would you like me to revert this so that it's not part of this PR?